packages/commons/src/auth/jwt.ts

```diff
@@ -24,7 +24,8 @@ export const readAuthDataFromJwtToken = (jwtToken: string): AuthData => {
 
 const getKey =
   (
-    clients: jwksClient.JwksClient[]
+    clients: jwksClient.JwksClient[],
+    loggerInstance: Logger
```

Comment:
could be just called `logger`, same for the rest of the file, to also reduce diff